core/src/test/java/com/graphhopper/storage/AbstractGraphStorageTester.java

```diff
@@ -36,7 +36,7 @@
 
 /**
  * Abstract test class to be extended for implementations of the Graph interface. Graphs
- * implementing GraphStorage should extend GraphStorageTest instead.
+ * implementing GraphStorage should extend {@link GraphHopperStorageTest} instead.
```

Comment:
here is no GraphStorageTest, but even with this change this javadoc seems outdated ?

Reply:
I think this is correct with your change. There were some implementations like for neo4j that used this abstract test, but all tests like GraphHopperStorageCHTest should not use it directly